input/about/bylaws.md

```diff
@@ -67,7 +67,17 @@ Amended and Restated Effective June 4, 2020
 
 **(e)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Limited Special Right for Director Appointed by Founding Member.** In connection with any vote to materially change the Foundation’s Membership Policy, Director Election Policy, Project Governance Policy, or any Intellectual Property-related agreements or policies, (a) a “no” vote by the director appointed by the Founding Member will result in the disapproval of a proposed action, regardless of the number of votes for approval), and (b) such director must be present as part of any quorum (i.e. if that director is not present, the Board will not have a quorum for the matter, regardless of the number of other directors present).
 
-**(f)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Notices.** All notices required under this Article III will be given to all directors in office at the time of such notice and may be given by email at least 48 hours in advance of the meeting or by first class mail to such director’s last known business address at least three business days in advance.
+**(f)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Member Review.** Prior to voting on any amendments to the Foundation’s Bylaws, Articles of Incorporation or the Foundation's Membership Policy, the Director Election Policy, the Project Governance Policy, or any of the Foundation’s Intellectual Property-related agreements or policies, the Board shall use reasonable efforts to preview the proposed amendments with the Foundation’s Members, using the following process:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(i)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;post the proposed amendments as “pull requests” on the Foundation’s Github, or through such other methodology determined by the Board that is accessible to Members for review and feedback;
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(ii)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;notify Members via a public announcement and solicit Member comments to the proposed changes over a designated Member review period (no less than a week from the date of notice), to be submitted by Members via Github or such other forum or methodology adopted by the Board;
```

Comment:
What kind of public announcement do do you see? In the current case it was announced from a private Twitter account (which was retweeted by official account). To have full transparency and get as much feedback as possible I would suggest to use email.

Reply:
Nitpicking a bit, but it was just any private twitter account, it was from the Executive Director's account. One could/should reasonable assume that news about the Foundation may come from there. And as you mention, it was retweeted by the Foundation's account too.

One concern over email is that sending out too many small announcements may be construed as spam by some users.

Reply:
Still in this case the exception is that Foundation members follow foundation accounts on Twitter and it's a catch or miss if they see the tweet. This might be easy for some which are active on Twitter, but probably will also exclude other members. Considering that diversity seems to be one of the main focus on .NET Foundation, this might lead to some people excluded in an important discussion.

Reply:
An option might be to have a separate repo for these changes, in which case it's possible for members if they want to watch the repo and actively receive notifications or not.

Reply:
Resolving as per my comment below with the dotnet-foundation/announcements repo.